code-with-comment.html

```diff
@@ -36,7 +45,25 @@
           type: Number,
           notify: true
         },
-        indexItem: Number
+        indexItem: Number,
+        hideAddComments: {
+          type: Boolean,
+          value: true
+        },
+        comments: Array,
+        hideComments: Boolean
+      },
+      _addHunk: function (e) {
+        this.hideAddComments = false;
+        e.stopPropagation();
+        var elem = e.detail;
+        elem.comments = this.comments;
+        this.fire('add-hunk-with-comment', elem);
+        this.comments = [];
+      },
+      _showAddComment: function () {
+        this.fire('toggle-add-comment');
+        this.hideAddComments = false;
```

Comment:
In the event handler above, you call `e.stopPropagation()` but here you do not. Why?